packages/client-ts/src/context/engines/verida/notification/engine.ts

```diff
@@ -27,15 +28,18 @@ export default class NotificationEngineVerida implements BaseNotification {
      /**
       * Ping a notification server to fetch new messages
       */
-    public async ping(): Promise<boolean> {
-        await this.init()
-        const server = await this.getAxios()
-
+    public async ping(config?: MessageSendConfig): Promise<boolean> {
+        await this.init();
+        const server = await this.getAxios();
+        
+        let context = config ? config.recipientContextName || this.context.getContextName() : this.context.getContextName();
+        let did = config ? config.did || this.did! : this.did!;
         try {
+            // Returns the client context and the corresponding `DID`
             await server.post(this.serverUrl + 'ping', {
```

Comment:
when I test, 
```
this.serverUrl + 'ping'
```
always miss a '/' character. for example `https://verida.ioping` instead of `https://verida.io/ping`

Reply:
@pkhien95 This will fix in a separate PR

Reply:
Fixed